packages/web/src/scss/components/Breadcrumbs/_theme.scss

```diff
@@ -12,7 +12,7 @@ $link-color-default: tokens.$action-link-primary-default;
 $link-color-active: tokens.$action-link-primary-active;
 $link-color-hover: tokens.$action-link-primary-hover;
 
-$current-link-color: tokens.$text-secondary-default;
+$current-link-color: tokens.$action-link-secondary-default;
```

Comment:
IMHO, this should be in a separate commit or PR.

Reply:
OK, I moved it here https://github.com/lmc-eu/spirit-design-system/pull/452